tests/store/model_registry/test_rest_store.py

```diff
@@ -55,7 +55,7 @@ def tearDown(self):
     def _args(self, host_creds, endpoint, method, json_body):
         res = {
             "host_creds": host_creds,
-            "endpoint": "/api/2.0/mlflow/%s" % endpoint,
+            "endpoint": "/api/2.0/preview/mlflow/%s" % endpoint,
```

Comment:
We should test `preview` and `regular` paths. If it's too much trouble, just leave a comment and we can get to it later.

Reply:
(discuss offline - client doesn't currently send requests to the preview paths so won't test this)